### ISSUE TITLE
Let tweet cards be True Dark as well

### ISSUE THREAD

**Issue by mikael-uhl** [j-fbriere/squawker#129]
### Describe the problem

When True Black is active, the cards still follows the current theme color. Since the cards take up so much space in the screen the true dark option is barely taking any effect on the feed.
E.g.: I have the "RedWine" theme active and True Dark enabled, when i open my feed each card background is tinted red, i appreciate the colored icons and buttons but not the card background.

### Describe the solution

Please let the background for tweet cards be true dark when TD is enabled. Hopefully it's as simple as changing the background color. Maybe adding a specific theme/themes for TD is also a solution.

**Comment by mrhcjones** [comment]
I have fixed this in the mentioned draft pull request above *No apk is available for testing at the moment*

**Comment by TheHCJ** [comment]
I just submitted an pull request to fix this issuef

**Comment by j-fbriere** [comment]
> This issue has been fixed and should be closed
It has been fixed in the last build, thank you for your help, but it has not been released yet.
So I would wait for the next release before closing it.

**Comment by j-fbriere** [comment]
The version [v3.8.0](https://github.com/j-fbriere/squawker/releases/tag/v3.8.0) has implemented this feature.

**Comment by j-fbriere** [comment]
On my Android when I have the System theme mode on is like having the Light theme mode on.
So for me the Cards color should not be true black.
Is your System theme mode like the Dark theme mode on your Android?

**Comment by k2helix** [comment]
For me it works when explicitly setting the Theme Mode to dark and in true dark mode. When the theme mode is set to system, the tweet cards are shown as always.

By the way, would it be too hard to make this optional? I think in some themes it may look better to have some contrast between the tweets and the background.

**Comment by TheHCJ** [comment]
Is this issue resolved @mikael-uhl?

**Comment by mikael-uhl** [comment]
> Is this issue resolved @mikael-uhl?

Tested on v3.8.2. "Theme Mode" is Dark and "True Black?" is checked.
The cards are still colored and not True Black, in comparison the background behind the cards is TB but it has been since before I've created this issue. However I did notice a diffence in colors, I think the cards are darker than previous versions but they are still tinted enough to light up AMOLED screens.
Since they are darker, I'm not sure how to proceed, was the intention just to darken the cards? Should I close this issue?
This kinda feature is a little hard for me to find in apps even today and it's why I appreciate the effort on making it work, it would be a big plus for me

App Info
"abis": [ "arm64-v8a", "armeabi-v7a", "armeabi" ]
"device": "miel", "flavor": "github", "locale": "en",
"os": "android",
"system": "33", "version": "300015940"
